### PR TITLE
Tc 2.0.5

### PR DESCRIPTION
**Pull request by f-peverali**
# Version Upgrade Template

Version: 2.0.5

Date:

## Description

This is a Pullreuqest that requires an increase in the Version number. Therefore, multiple outside-github, related Task have to be performed and checked.

All jobs with an `x` in the boxes were performed to the best of knowledge.

## Pre-Merge Activities

- [x] This PR refers to a versioned Branch with a name and a version number in the form of N.n.n, e.g. "TC_3.2.1".
- [x] This PR has a clean meaningful committ history. Minor committs or committs without descirption have been squashed, at the latest now.
- [x] The _./github/workflows/main.yml_ refers to the correct Firetly Terminal and SUSHI Version. 
    > **_Firely Terminal Pipeline_**  0.4.0.

    > **_SUSHI Versions_**  3.5.0.
- [x] By running the _Release_Publish.py_ script, release version and date was updated accordingly. The script ran without errors.
- [x] Eventually, increase the dependency of to newer Basis Modul (package and sushi-config)
- [x] New Release Notes were created, aglined to the committ history and cleaned. In Github, go to 
  - [x]  _-> Releases_ then _-> Draft a new release_ with the _Modul Name and Version_, then
  - [x] _-> Target the main-Branch_ and _->enter a new Tag according to the Version_, then click.
  - [x] Click _-> Generate Release notes_ , _->Adjust them if necessary_ and _-> Copy/Paste the Details in the RealeaseNotes.md_ of the very Branch you want to merge.
  - [x] Finally _-> Save as Draft_

## Merge and Publishing

- [x] With the updated Version, Dates, and Release Notes (as described above) with the last committ into the Branch you want to merge.
- [x] In GitHub _-> Actions_ the _->CI (FHIR Validation)_ workflow terminates successfully.
- [x] Add the Approve / the PR gets posivitly reviewed by a collegue.
- [x] Merge (without squash) the PR, delete the Branch.

## Post-Merge Activities

- [x] Go to the corresponding SIMPLIFIER Project and _-> Github -> Reimport_ the project.
- [x] Go to the corresponding SIMPLIFIER Project and _-> Packages -> Expand the Dropdown for Create -> Create new package_ for the project.
  - [x] With the corresponding version number, and
  - [x] The Release notes (from above) and a compare-link to the previous Release.
  - [x] Unlist the old package by _-> clicking on the old package_, _-> go to Admininstration_ and _-> click on Unlist_
- [x] Publish the previosuly drafteted Release, including version number, on GitHub.
- [x] Provide / Archive the IG in the corresponding _gh-pages_ branch of the GitHub project.
  - [x] Checkout the Branch (no need to merge it later).
  - [x] Export from Simplifier via _-> Guides -> Expand the Modul ... -> Export_
  - [x] Unpack the zip, remove the packages folder (because its kinda big), and move everything else to a (version coressponding) new folder in the branch folder structure.
  - [x] committ the branch.

### CODE REVIEW

File: Resources/input/fsh/ISiKKalender.fsh [hunk 1]
```diff
@@ -8,8 +8,16 @@ Id: ISiKKalender
...
   * ^slicing.discriminator.path = "$this"
   * ^slicing.rules = #open
-* specialty contains Fachrichtung 1..1 MS
-* specialty[Fachrichtung] from $authorSpecialtyVS (required)
+* specialty contains 
+  Fachrichtung 1..1 MS and 
+  Fachspezialisierung 0..1 and 
+  ErweiterterFachabteilungsschluessel 0..1
+* specialty[Fachrichtung] from $IHEpracticeSettingVS (required)
+  * ^comment = "Die Wahl des hinterlegten ValueSets (http://ihe-d.de/ValueSets/IHEXDSpracticeSettingCode) wurde mit einem Mitglied der IHE Deutschland Arbeitsgruppe XDS ValueSets (https://www.ihe-d.de/projekte/xds-value-sets-fuer-deutschland/) abgestimmt (Stand:13.06.2024)."
```
Comment (f-peverali) [inline Resources/input/fsh/ISiKKalender.fsh]: issue: hier ist noch eine klarstellung zur Optionalität des MUST Support (aufgrund der späten Änderung notwednig)

File: Resources/input/fsh/ISiKTermin.fsh [hunk 2]
```diff
@@ -45,8 +45,19 @@ Id: ISiKTermin
...
+  Fachspezialisierung 0..1 and 
+  ErweiterterFachabteilungsschluessel 0..1
+* specialty[Fachrichtung] from $IHEpracticeSettingVS (required)
+  * ^definition = "Conditional Must Support - Einschränkung der übergreifenden MS-Definition: ein bestätigungsrelevantes System SOLL das ValueSet (http://ihe-d.de/ValueSets/IHEXDSpracticeSettingCode) implementieren, MUSS es jedoch NICHT."
+  * ^comment = "Hintergrund zum MS: die MS-Änderung erfolgt als Technical Correction in Stufe 2 spät während der Implementierungsphase, daher ist die Abbildung dieses ValueSets nicht verpflichtend.
+
+  Hintergrund zum ValueSet: Die Wahl des hinterlegten ValueSets (http://ihe-d.de/ValueSets/IHEXDSpracticeSettingCode) wurde mit einem Mitglied der IHE Deutschland Arbeitsgruppe XDS ValueSets (https://www.ihe-d.de/projekte/xds-value-sets-fuer-deutschland/) abgestimmt (Stand:13.06.2024)."
+* specialty[Fachspezialisierung] from $authorSpecialtyVS (required)
+  * ^comment = "Dieses Slice SOLL NICHT genutzt werden und ist nur aufgrund der Kompatibilität beibehalten worden (Stand:13.06.2024)."
+* specialty[ErweiterterFachabteilungsschluessel] from $FachabteilungsschluesselErweitertCS (required)
```
Comment (alexey-tschudnowsky) [inline Resources/input/fsh/ISiKTermin.fsh]: issue: $FachabteilungsschluesselErweitert**V**S statt $FachabteilungsschluesselErweitert**C**S

File: Resources/input/fsh/ISiKTermin.fsh [hunk 3]
```diff
@@ -45,8 +45,19 @@ Id: ISiKTermin
   * ^slicing.discriminator.type = #pattern
   * ^slicing.discriminator.path = "$this"
   * ^slicing.rules = #open
-* specialty contains Fachrichtung 1..1 MS
-* specialty[Fachrichtung] from $authorSpecialtyVS (required)
+* specialty contains 
+  Fachrichtung 1..1 MS and
```
Comment (alexey-tschudnowsky) [inline Resources/input/fsh/ISiKTermin.fsh]: question / issue: wiederspricht die 1..1 Definition dem unteren Kommentar "daher ist die Abbildung dieses ValueSets nicht verpflichtend"?

File: Resources/input/fsh/ISiKTermin.fsh [hunk 4]
```diff
@@ -45,8 +45,19 @@ Id: ISiKTermin
...
+  * ^comment = "Hintergrund zum MS: die MS-Änderung erfolgt als Technical Correction in Stufe 2 spät während der Implementierungsphase, daher ist die Abbildung dieses ValueSets nicht verpflichtend.
+
+  Hintergrund zum ValueSet: Die Wahl des hinterlegten ValueSets (http://ihe-d.de/ValueSets/IHEXDSpracticeSettingCode) wurde mit einem Mitglied der IHE Deutschland Arbeitsgruppe XDS ValueSets (https://www.ihe-d.de/projekte/xds-value-sets-fuer-deutschland/) abgestimmt (Stand:13.06.2024)."
+* specialty[Fachspezialisierung] from $authorSpecialtyVS (required)
+  * ^comment = "Dieses Slice SOLL NICHT genutzt werden und ist nur aufgrund der Kompatibilität beibehalten worden (Stand:13.06.2024)."
+* specialty[ErweiterterFachabteilungsschluessel] from $FachabteilungsschluesselErweitertCS (required)
+  * ^comment = "Dieses ValueSet KANN über ein Mapping (siehe Abschnitt https://wiki.hl7.de/index.php?title=IG:Value_Sets_f%C3%BCr_XDS#DocumentEntry.practiceSettingCode) mit dem ValueSet der Fachrichtung verknüpft werden und darüber ggf. die Integration von Systemen erleichtern."
```
Comment (alexey-tschudnowsky) [inline Resources/input/fsh/ISiKTermin.fsh:60]: suggestion: in die unteren Beispiele aufnehmen

File: Resources/input/fsh/aliases.fsh [hunk 5]
```diff
@@ -1,4 +1,8 @@
 Alias: $appointmentStatus = http://hl7.org/fhir/appointmentstatus
 Alias: $cancelationReason = http://terminology.hl7.org/CodeSystem/appointment-cancellation-reason
 Alias: $authorSpecialtyVS = http://ihe-d.de/ValueSets/IHEXDSauthorSpeciality
-Alias: $capabilitystatement-expectation = http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation
\ No newline at end of file
+Alias: $capabilitystatement-expectation = http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation
+Alias: $IHEpracticeSettingVS = http://ihe-d.de/ValueSets/IHEXDSpracticeSettingCode
```
Comment (alexey-tschudnowsky) [inline Resources/input/fsh/aliases.fsh:5]: question / issue: entsteht hier eine Package-Abhängigkeit, z.B. zu [de.ihe-d.terminology 3.0.0](https://simplifier.net/packages/de.ihe-d.terminology/3.0.0)?

File: Resources/input/fsh/ISiKTermin.fsh (hunk 2 above)
Comment (f-peverali) [reply Resources/input/fsh/ISiKTermin.fsh]: done

File: Resources/input/fsh/ISiKTermin.fsh (hunk 3 above)
Comment (f-peverali) [reply Resources/input/fsh/ISiKTermin.fsh]: Kardinalität hier in Stufe 2 muss 0..1 sein. DONE

File: Resources/input/fsh/aliases.fsh (hunk 5 above)
Comment (f-peverali) [reply Resources/input/fsh/aliases.fsh:5]: dependency wird hinzugefügt

File: Resources/input/fsh/ISiKKalender.fsh (hunk 1 above)
Comment (f-peverali) [reply Resources/input/fsh/ISiKKalender.fsh]: done

**Pull request merged**